### ISSUE TITLE
How do you use a library? (newbye)

### ISSUE THREAD

**Issue by vinniec** [extremeheat/JSPyBridge#95]
I would like to try to do the search with [this library](https://github.com/facundoolano/google-play-scraper#search)  but I don't understand exactly how to do it:
```
from javascript import require
scrjs = require("google-play-scraper")
scrjs['search']({"term":"panda", "num": 250})
```
1) how to capture the output, the resulting list?
2) I don't understand much of this but I think the response might be asynchronous so I'm not getting all the results? How to get them all?

**Comment by extremeheat** [comment]
Hi,

Your code above seems to work fine. I created a notebook on Google Colab to test your code you can view here https://colab.research.google.com/gist/extremeheat/a76b258a4e7cd7c08f1b6303b9947324

**Comment by vinniec** [comment]
I'm sorry for the delay but I was traveling and I got distracted a few days.
I had used this js library a few years ago and it worked, however I don't chew javascript and I consider quite difficult for me doing what I do very quickly in python.
So I wanted to try again with your module and was strange seeing that I got no more than 30 results and I thought it was a problem with your module.
Since you told me that it works instead I checked and it seems that you are right:
https://github.com/facundoolano/google-play-scraper/issues/642
https://github.com/facundoolano/google-play-scraper/issues/527
There is some problem with the pages changes, unfortunately, it was the only library I knew that returned more than 30 results.

Anyway thank you very much for the example, I didn't know colab, I always used jupyter only locally.
However your example is not exactly the same as mine, I had tried to invoke scrjs.search() while you treated it like a dictionary scrjs['search'.
I really understand very little about js though but the strangest thing to me is that they have different outputs.
